src/index.html

```diff
@@ -15,8 +16,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <!--#endregion -->
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img
+        class="logo__image"
+        src="images/logo.png"
+        alt="logo_image"
+      />
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__list__item">
+            <a
+              href="http://google.com"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              href="http://google.com"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              href="http://google.com"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="nav__list__item"
+            data-qa="hover"
+          >
+            <a
+              href="http://google.com"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              href="http://google.com"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              href="http://google.com"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              href="http://google.com"
```

Comment:
The `href` attribute is set to 'http://google.com'. If this is a placeholder, consider updating it to the correct URL or using a relative path if it's an internal link.